lib/idpartner.js

```diff
@@ -166,6 +214,8 @@ class IDPartner {
       'x-fapi-interaction-id': uuidv4(),
       identity_provider_id: query.idp_id,
       idpartner_token: idpartnerToken,
+      claims: JSON.stringify(claimsObject),
+      claims_mapped_to_scopes: claimsMappedToScopes.join(' '),
```

Comment:
Unsure why this is needed. Let me review the complementary PRs and I will potentially auto-resolve this comment. Ignore for now.

Reply:
I would advise to send to the integration-service only the scope and claims params. Then, let it do whatever it has to do to compute and get the correct claims/scopes.